modulemd/tests/ModulemdTests/modulestream.py

```diff
@@ -586,8 +586,17 @@ def test_v3_dependencies(self):
         assert stream.get_runtime_requirement_stream("testmodule") == "latest"
 
     def test_xmd(self):
-        if "_overrides_module" in dir(Modulemd):
-            for version in modulestream_versions:
+        for version in modulestream_versions:
+            # We have a chicken-egg problem with overrides, since they can only
+            # be tested if they are already installed. This means they need to
+            # be run in the CI. In order to avoid changes to these tests or the
+            # overrides breaking things, we'll skip them if the appropriate
+            # override is not installed.
+            if hasattr(
```

Comment:
```suggestion
            if "_overrides_module" in dir(Modulemd) and hasattr(
```
This should make the tests happier.